### DISCUSSION TITLE
Add Categories

### DISCUSSION THREAD

**Discussion by Araz0** [motcodes/recommends#5] [Feature Request]
Having some kind of a Category system on the index page would help tremendously while searching for the topic a person is looking for.
Languages like PHP RoR and others can be under the web server-side category for example (hopefully a shorter category name).
A combination of the title and icon of each topic and the Category system should speed up the process of finding the desired topic page and content.

**Comment by motcodes** [comment]
Ok the main problem is that a category can't be to specific. Let's take a basic google search query for example. The way (at least how I search my stuff is) is: styled-components ssr next js. 
So Topic, Category and a specific keyword to narrow it down. 

Therefore you would look for the Topic first and then the Category.
